src/Ormolu/Diff/ParseResult.hs

```diff
@@ -7,6 +7,7 @@
 {-# LANGUAGE RecordWildCards #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE ViewPatterns #-}
+{-# OPTIONS_GHC -Wno-incomplete-record-updates #-}
```

Comment:
Why is this necessary?

Reply:
Due to this code below:
```haskell
classDeclCtxEq :: TyClDecl GhcPs -> GenericQ ParseResultDiff
classDeclCtxEq tc@ClassDecl {tcdCtxt = Just (L _ [])} tc' = genericQuery tc {tcdCtxt = Nothing} tc'
classDeclCtxEq tc tc' = genericQuery tc tc'
```
But it just occurred to me that this could be rewritten using `RecordWildCards` without an incomplete record update.